docs/reference/class/MongoDBBulkWriteResult.txt

```diff
@@ -0,0 +1,26 @@
+.. _php-bulk-write-result:
+
+==============================
+MongoDB\\BulkWriteResult Class
```

Comment:
If all of these pages are nestled under "API Reference", I suggest we just get rid of the " Class" suffix to conserve space in the menus. It's redundant and tends to add extra word wrapping to the menu.

I'll defer to @alcaeus and @GromNaN, though.

Reply:
I don't think it's a problem since it's on the right. And it's a bit more explicit.